README.md

```diff
@@ -34,6 +34,15 @@ If you want to add Discourse links, you must also configure the prefix for your
                     "discourse_prefix": "https://discuss.linuxcontainers.org/t/"
                    }
 
+You can configure different Discourse prefixes by specifying a dict:
+
+    html_context = {
+                    (...),
+                    "discourse_prefix": {
+                        "lc": "https://discuss.linuxcontainers.org/t/",
```

Comment:
```suggestion
                        "lxc": "https://discuss.linuxcontainers.org/t/",
```
That is because lxc is the short form of linux containers.